apps/app.go

```diff
@@ -45,23 +44,20 @@ func (id AppId) String() string {
 	return string(id)
 }
 
-func (id AppId) ConsulServiceName() string {
-	return strings.Replace(strings.Trim(strings.TrimSpace(id.String()), "/"), "/", ".", -1)
-}
-
 func (app *App) IsConsulApp() bool {
 	_, ok := app.Labels[MARATHON_CONSUL_LABEL]
 	return ok
 }
 
-func (app *App) ConsulServiceName() string {
-	if value, ok := app.Labels[MARATHON_CONSUL_LABEL]; ok && value != "true" {
-		value = AppId(value).ConsulServiceName()
-		if value != "" {
-			return value
-		}
+func (app *App) ConsulName() string {
```

Comment:
If we are moving Consul related logic from here this could be also moved to consul.